backends/magma/kernels/common/grad_device.h

```diff
@@ -335,4 +331,4 @@ magma_grad_generic_device(
 #undef C
 }
 
-#endif    // MAGMA_BASIS_APPLY_GRAD_DEVICE_CUH
+#endif    // MAGMA_BASIS_APPLY_GRAD_DEVICE_H
```

Comment:
generally, the internal header guards for the rest of the library are prefaced with `_ceed_`

Reply:
I added CEED to all the header guards in 5debdf81ca2cd4ab28cea5645132790c3091e889